RELEASE.md

```diff
@@ -0,0 +1,31 @@
+
+### Creating a release
+
+Scispacy has two components:
+
+- The scispacy pip package
+- The scispacy models
+
+The scispacy pip package is published automatically using the `.github/actions/publish.yml` github action. It happens whenever a release is published (with an associated tag) in the github releases UI.
+
+In order to create a new release, the following should happen:
+
+#### Updating `scispacy/version.py`
+Update the version in version.py.
+
+#### Training new models
+
+For the release, new models should be trained using the `scripts/pipeline.sh` and `scripts/ner_pipeline.sh` scripts, for the small, medium and large models, and specialized NER models. Remember to export the `ONTONOTES_PATH` and `ONTONOTES_PERCENT` environment variables to mix in the ontonotes training data.
```

Comment:
also, note that we start the NER models from the medium base model, not a blank model